examples/huggingface-transformers/README.md

```diff
@@ -14,137 +14,117 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Hugging Face Transformers - DeepSparse Inference Examples
-This directory contains examples for serving, benchmarking, and running NLP 
-models from the [Hugging Face Transformers](https://github.com/huggingface/transformers)
-repository using the DeepSparse Engine. These examples can load pre-trained,
-sparsified models from [SparseZoo](https://github.com/neuralmagic/sparsezoo) 
-or you can specify your own transformer [ONNX](https://onnx.ai/) file.
-
-## Installation
-The dependencies for these examples can be installed using `pip` and the supplied `requirements.txt` file:
-```bash
-pip3 install -r requirements.txt
-```
+# Transformers 🤗 Inference Pipelines
+This directory contains examples for serving, benchmarking, and running NLP models from the [Transformers](https://github.com/huggingface/transformers) repository using the DeepSparse Engine. These examples can load pre-trained, sparsified models from SparseZoo or you can specify your own transformer ONNX file. In addition, we also highlight how you can easily perform benchmarking and deploy transformers with the `deepsparse.server` via simple CLI commands.
 
-## DeepSparse Pipeline Example
+### Installation
 
-The DeepSparse-Hugging Face pipeline integration provides a simple API 
-dedicated to several tasks,
-following is an example using a pruned BERT model from the SparseZoo for 
-Question-Answering task. Other currently supported tasks include
-`text-classification`, `sentiment-analysis`, `ner`, and `token-classification`.
+```bash
+pip install deepsparse
+```
 
+The DeepSparse-Hugging Face pipeline integration provides a simple API dedicated to several tasks:
+### Token Classification Pipeline
 ```python
 from deepsparse.transformers import pipeline
 
-# SparseZoo model stub or path to ONNX file
-onnx_filepath="zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-aggressive_98"
-
-num_cores=None  # uses all available CPU cores by default
+model_path = "zoo:nlp/token_classification/bert-base/pytorch/huggingface/conll2003/base-none"
 
-# Get DeepSparse question-answering pipeline
-
-qa_pipeline = pipeline(
-    task="question-answering",
-    model_path=onnx_filepath,
-    num_cores=num_cores,
+token_classification = pipeline(
+    task="token-classification",
+    model_path=model_path,
+    num_cores=None
 )
 
-# inference
-
-my_name = qa_pipeline(question="What's my name?", context="My name is Snorlax")
+inference = token_classification("I saw Snorlax in Texas!")
```

Comment:
what will this actually do/classify?

Reply:
it will print a list of dictionaries. Each dict will represent the classification of a token 
example of a single dict:
{'entity': 'LABEL_0', 'score': 0.9997255, 'index': 1, 'word': 'i', 'start': 0, 'end': 1}

One thing I want to point out, is that the `entity` variable doesn't have the label of the actual entity. This would be good if we can get these actual entities in the print out.